client/my-sites/plans-features-main/index.jsx

```diff
@@ -243,15 +241,15 @@ class PlansFeaturesMain extends Component {
 					answer={ translate(
 						'No. All WordPress.com sites include our specially tailored WordPress hosting to ensure' +
 						' your site stays available and secure at all times. You can even use your own domain' +
-						' when you upgrade to the premium or business plan.'
+						' when you upgrade to the Personal, Premium, or Business plan.'
 					) }
 				/>
 
 				<FAQItem
 					question={ translate( 'Do you offer email accounts?' ) }
 					answer={ translate(
-						'Yes. If you register a new domain with our premium or business plans, you can optionally' +
-						' add G Suite. You can also set up email forwarding for any custom domain' +
+						'Yes. If you register a new domain with our Personal, Premium, or Business plans, you can optionally' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Yes. If you register a new domain with our premium or business plans, you can optionally add G Suite. You can also set up email forwarding for any custom domain registered through WordPress.com. {{a}}Find out more about email{{/a}}.' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).